### PR TITLE
Bump version in main

### PR DESCRIPTION
**Pull request by azdagron**

### CODE REVIEW

**Review by MarcosDY — DISMISSED** [review]
LGTM!!!

File: test/integration/suites/upgrade/versions.txt [hunk 1]
```diff
@@ -5,3 +5,5 @@
 1.4.0
 1.4.1
 1.4.2
+1.4.3
```
Comment (amartinezfayo) [inline test/integration/suites/upgrade/versions.txt:8]: 1.3.x versions should be removed here.

Comment (azdagron) [reply test/integration/suites/upgrade/versions.txt:8]: That doesn't happen until after we release 1.5.0 and add it here, right?

Comment (azdagron) [reply test/integration/suites/upgrade/versions.txt:8]: Now I need to look to see what we've done in the past :rofl:

Comment (azdagron) [reply test/integration/suites/upgrade/versions.txt:8]: Ok, I re-read our comment in the upgrade IT README and you're right

**Review by amartinezfayo — APPROVED** [review]
🎉

**Pull request merged**